vexim/adminaliasaddsubmit.php

```diff
@@ -52,6 +52,10 @@
     $dbh,$_POST['localpart'],$_SESSION['domain_id'],'adminalias.php'
   );
 
+  if($_POST['realname']==='') {
```

Comment:
I would probably also check for `!isset()`, even if knowing that this field is always being submitted.